src/Categories/CategoryEntity.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace FurnitureStoreApi\Categories;
+header('Content-Type: application/json; charset=utf-8');
+
+class CategoryEntity implements \JsonSerializable
+{
+    private int $id;
+    private string $category;
+    private int $category_id;
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function getCategory(): string
+    {
+        return $this->category;
+    }
+
+    public function getCategoryid(): int
```

Comment:
Is this really returning the categoryID, or is it actually the count of products in that category?